app/install/dcos.md

```diff
@@ -43,30 +44,30 @@ have basic knowledge of [DC/OS]({{ page.links.dcos }}),
     [DC/OS CLI]({{ page.links.mesos-cli }}) or the
     [DC/OS GUI]({{ page.links.mesos-gui }}).      
 
-3. **Deploy Marathon-lb**
+3. **Deploy Marathon-LB**
 
-    Using the `marathon-lb-internal.json` file included in the kong-dist-mesos
-    repo, deploy Marathon-lb for internal and external service discovery
-    support:
+    We will use [Marathon-LB]({{ page.links.marathon-lb }}) for load balancing
+    N/S traffic to cluster and [VIPs]({{page.links.vips}}) for load balancing
+    E/W traffic. Using the package `marathon-lb` deploy Marathon-LB for
+    external load blance support and service discovery:
```

Comment:
The above paragraph doesn't make sense.

> We will use Marathon-LB .... Using the package

Needs a sentence break to introduce the package after denoting usage.

``` We will use the `marathon-lb` package to deploy [Marathon-LB]({{ page.links.marathon-lb }}) for load balancing,  and N/S traffic to cluster and [VIPs]({{page.links.vips}}) for load balancing E/W traffic.```